backend/inspirehep/assign/views.py

```diff
@@ -109,3 +109,21 @@ def literature_assign_conferences_view(args):
         return jsonify({"message": "Internal Error"}), 500
 
     return jsonify({"message": "Success"}), 200
+
+
+@blueprint.route("literature-cds", methods=["POST"])
+@login_required_with_roles([Roles.cataloger.value])
+@parser.use_args(
+    {
+        "literature_recids": fields.List(fields.Integer, required=True),
+    },
+    locations=("json",),
+)
+def literature_export_to_cds(args):
+    literature_recids = args["literature_recids"]
+    try:
+        export_paper_to_cds(literature_recids)
+    except Exception as ex:
+        LOGGER.exception("Cannot start 'export_to_cds' task.", err=ex)
```

Comment:
It automatically get the exception
```suggestion
        LOGGER.exception("Cannot start 'export_to_cds' task.")
```

Reply:
Could we fix it also in `literature_assign_conferences_view`? 🤓